components/server/src/authorization/authorizer.ts

```diff
@@ -24,13 +24,14 @@ import {
 } from "./definitions";
 import { SpiceDBAuthorizer } from "./spicedb-authorizer";
 import { getExperimentsClientForBackend } from "@gitpod/gitpod-protocol/lib/experiments/configcat-server";
+import { log } from "@gitpod/gitpod-protocol/lib/util/logging";
 
 export function createInitializingAuthorizer(spiceDbAuthorizer: SpiceDBAuthorizer): Authorizer {
     const target = new Authorizer(spiceDbAuthorizer);
     const initialized = (async () => {
         await target.addInstallationAdminRole(BUILTIN_INSTLLATION_ADMIN_USER_ID);
         await target.addUser(BUILTIN_INSTLLATION_ADMIN_USER_ID);
-    })();
+    })().catch((err) => log.error("Failed to initialize authorizer", err));
```

Comment:
this would hide the error and only log. Is that intentional?

Reply:
Whoops, good catch.